src/discord-cluster-manager/cogs/github_cog.py

```diff
@@ -35,18 +35,22 @@ async def run_github(
         interaction: discord.Interaction,
         script: discord.Attachment,
         gpu_type: app_commands.Choice[str],
-    ):
+        use_followup: bool = False
+    ) -> discord.Thread:
         if not script.filename.endswith(".py") and not script.filename.endswith(".cu"):
             await interaction.response.send_message(
                 "Please provide a Python (.py) or CUDA (.cu) file"
             )
-            return
+            return None
 
         thread = await self.bot.create_thread(interaction, gpu_type.name, "GitHub Job")
+        message = f"Created thread {thread.mention} for your GitHub job"
+
+        if use_followup:
+            await interaction.followup.send(message)
```

Comment:
you might have answered this already but remind me what does followup mean?

Reply:
So interactions can be replied to with `interaction.response.send_message(...)`. But in this case that's the last message that the bot can send for the interaction. And the other option is to use `interaction.followup.send(...)`, which lets multiple messages be sent in response to the interaction. 

As to why this distinction exists, I believe the reason is that interactions need timely responses -- within 3 seconds. So if the response is going to take longer than that, as it does with training runs, then followups have to be used.

By the way, VerifyRunCog has to set up the interaction for followups by calling `interaction.response.defer()`.

Hope that explains things. LMK if you have more questions!